src/app/candidate-management/candidate-management.module.ts

```diff
@@ -24,32 +24,12 @@ export const CANDIATE_ROTUES = [
         component: CandidateManagementListComponent,
       },
       {
-        path: 'detail/:name',
+        path: ':action',
```

Comment:
This can only be the create action? If so (and if the create action has its own component) then the create action can probably be hardcoded here.

Reply:
There can only be the create action, however if I hardcode it and don't save it into the "action" variable, the component does not know if it is called to create a new candidate or if it was called with an invalid action (or "edit" oder "detail")